pages/home.html

```diff
@@ -100,10 +106,8 @@
     Contributing to Oppia
   </h2>
   <p>
-    Oppia is built by the community for the community. Our mission is to enable
-    anyone to learn anything they want to in an effective, enjoyable way, and
-    building a platform that allows anyone to create and share bite-sized
-    interactive learning material is just a first step.
+    <em>Built by the community for the community</em> – Oppia aims to enable
```

Comment:
I like it as a motto, but we don't really use this construct (headline in italics) anywhere else on the page.

How about putting it in italics? "Oppia is _built by the community for the community_ -- our mission is to ..."